pandas/tests/interchange/test_impl.py

```diff
@@ -417,17 +416,35 @@ def test_non_str_names_w_duplicates():
         pd.api.interchange.from_dataframe(dfi, allow_copy=False)
 
 
-@pytest.mark.parametrize(
-    "dtype", ["Int8", pytest.param("Int8[pyarrow]", marks=td.skip_if_no("pyarrow"))]
-)
-def test_nullable_integers(dtype: str) -> None:
+def test_nullable_integers() -> None:
+    # https://github.com/pandas-dev/pandas/issues/55069
+    df = pd.DataFrame({"a": [1]}, dtype="Int8")
```

Comment:
Could you include a `None` in here to test null?

And parametrize over `["Int8", "Float32", "boolean"]`?

Reply:
boolean isn't supported yet, there's a separate issue about that: https://github.com/pandas-dev/pandas/issues/55332

sure, I've added a separate test `test_pandas_nullable_w_missing_values` which includes a null